src/redux/store.ts

```diff
@@ -1,22 +1,35 @@
+import { initDrizzle } from '../blockchain/drizzle';
 import { applyMiddleware, compose, createStore } from 'redux';
 import createSagaMiddleware from 'redux-saga';
+import { generateContractsInitialState } from 'drizzle';
 
+import RockPaperScissorsGame from '@contracts/RockPaperScissorsGame.json';
```

Comment:
These references will fail until truffle:migrate is run. Not sure what the best way around that is.

Reply:
I think that's ok, it just needs to be part of the build process, right?

The annoying thing is that the contracts can't easily be watched and recompiled, but we won't often want to change the contracts (I hope)

Reply:
We also need to change the build process to strip out unnecessary information from the contracts, for when we actually want to deploy the app.

Reply:
👍